src/lib.rs

```diff
@@ -450,6 +476,22 @@ impl Build {
         self
     }
 
+    /// Set CUDA C++ support.
+    ///
+    /// Enabling CUDA will pass the detected C/C++ toolchain as an argument to
+    /// the CUDA compiler, NVCC. NVCC itself accepts some limited GNU-like args;
+    /// any other arguments for the C/C++ toolchain will be redirected using
+    /// "-Xcompiler" flags.
+    ///
+    /// If enabled, this also implicitly enables C++ support.
+    pub fn cuda(&mut self, cuda: bool) -> &mut Build {
+        self.cuda = cuda;
+        if cuda {
+            self.cpp(true);
```

Comment:
I'm pretty unknowledgeable when it comes to everything cuda, but is there maybe C code which gets compiled as well? Or does this suffice for all compilations?

Reply:
AFAIK the "CUDA" dialect is really a superset of C++11 (with extensions for writing and calling GPU code in something that looks like C++). Although NVCC should be able to indirectly compile plain C files if you pass it `-ccbin=gcc`.

Reply:
Ok makes sense!